FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -367,16 +313,12 @@ - (FIRRemoteConfigValue *)objectForKeyedSubscript:(NSString *)key {
 }
 
 - (FIRRemoteConfigValue *)configValueForKey:(NSString *)key {
-  return [self configValueForKey:key namespace:_FIRNamespace];
-}
-
-- (FIRRemoteConfigValue *)configValueForKey:(NSString *)key namespace:(NSString *)aNamespace {
+  NSString *aNamespace = _FIRNamespace;
   if (!key || !aNamespace) {
     return [[FIRRemoteConfigValue alloc] initWithData:[NSData data]
                                                source:FIRRemoteConfigSourceStatic];
   }
   NSString *FQNamespace = [self fullyQualifiedNamespace:aNamespace];
```

Comment:
Do we still need to do this? Or can we assume that `_FIRNamespace` is always nonnull and fully qualified?

Reply:
`key` is nullable so that test still needs to be there.

Yes it looks like _FIRNamespace is always set up in the initializer, so you could get rid of the `aNamespace` local and test.

Reply:
That makes sense, removed `aNamespace`